ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
According to [this thread](http://stackoverflow.com/a/29046154) the ContentType is not removed by the migration actually. We should make sure we run migrate with `--no-input` btw.